yarl/_path.py

```diff
@@ -31,11 +34,32 @@ def normalize_path_segments(segments: Sequence[str]) -> list[str]:
 def normalize_path(path: str) -> str:
     # Drop '.' and '..' from str path
     prefix = ""
-    if path and path[0] == "/":
+    if path and path[0] == SEPARATOR:
         # preserve the "/" root element of absolute paths, copying it to the
         # normalised output as per sections 5.2.4 and 6.2.2.3 of rfc3986.
-        prefix = "/"
+        prefix = SEPARATOR
```

Comment:
Please leave the these as is. It's more confusing to have them as a name constant especially since the url spec has multiple different separators.